assets/js/modules/analytics-4/datastore/api.js

```diff
@@ -52,17 +50,7 @@ const baseSelectors = {
 			}
 		}
 
-		for ( const propertyList of Object.values( state.properties ) ) {
-			if ( propertyList && propertyList.length && propertyList.length > 0 ) {
-				for ( const webdatastreamList of Object.values( state.webdatastreams ) ) {
-					if ( webdatastreamList && webdatastreamList.length && webdatastreamList.length > 0 ) {
-						return true;
-					}
-				}
-			}
-		}
-
-		return undefined;
+		return true;
```

Comment:
See my other comment, this still needs to be able to differentiate between `undefined` and `true`. The logic you removed here indeed looked incorrect though.

What would be more appropriate here would be to:
* If any `getProperties` or `getWebDataStreams` (or `getAccountSummaries`) selector resolution resulted in an error, it should return `false`.
* Otherwise, if any `getProperties` or `getWebDataStreams` (or `getAccountSummaries`) selector has finished resolution, it should return `true`.
* Otherwise, it should return `undefined`.